POS/UserInterface/01a_Cashpoint.md

```diff
@@ -9,82 +9,139 @@
+- *Find offer by name* ![Search](../../Assets/Icons/Search.png "[Search]")   
+    Enter the offer name, the keyword or the (EAN) number you search for in the search bar. Click an offer in the list of search results to add it to the bill list.
 
     > [Info] The search already provides you with proposals for your keyword as you type.
 
 - *QuickSelect Category*    
-    Click a QuickSelect category tab to display the associated QuickSelect elements. The tabs are only displayed if a QuickSelect category with an element has been created before. The categories are named individually.
+    Click a QuickSelect category tab to display the associated QuickSelect elements. The tabs are only displayed if a QuickSelect category with an element has been created before. The categories are named individually. 
 
 - *QuickSelect Element*   
-    Click a QuickSelect element to add it to the bill list. The QuickSelect element is only displayed if a QuickSelect category with an element has been created before. The elements are named individually.
+    Click a QuickSelect element to add it to the bill list. The QuickSelect element is only displayed if a QuickSelect category with an element has been created before. The elements are named individually.  For detailed information, see [Create a QuickSelect Category](../Integration/07_ManageOffers.md#create-a-quickselect-category).
+
+
+<!---NEU-->
+## Age verification
+
+*Venduo POS > Sales > Select Store and Pay Desk > Tab CASHPOINT > Select offer*
+
+![Age verification](../../Assets/Screenshots/POS/Sales/Cashpoint/AgeVerification.png "[Age verification]")
+
+The *Age verification* window is displayed if a selected offer is not suitable for children and young people. For detailed information, see [Verify the age](../Operation/02_SelectOffer.md#verify-the-age).
+
+- [AGE OF BUYER CONFIRMED]   
+    Click this button if the customer is allowed to buy this offer.  
+    The offer is added to the bill list.      
+
+- [CANCEL]   
+   Click this button if the customer is too young to buy this offer.   
+   The offer is not added to the bill list. 
+
+
+
+<!---NEU-->
+## Choose options
+
+*Venduo POS > Sales > Select Store and Pay Desk > Tab CASHPOINT > Select offer*
+
+![Choose options](../../Assets/Screenshots/POS/Sales/Cashpoint/ChooseOption2.png "[Choose option]")
+
+The *Choose options* window is displayed if a selected offer has additional product options that represent a service. For detailed information, see [Add a product option](../Operation/02_SelectOffer.md#add-a-product-option). 
+
+- Additional field   
+    Enter additional information to a single option, if applicable. This can be a text or a name that should be printed on a shirt or another item.
+
+- [CANCEL]   
+    Click this button if the customer no longer wants to purchase the offer associated with the product option.
```

Comment:
Ich würde nicht so konkret beschreiben... ("if the customer no longer wants to..." Wir wissen nicht, ob er will oder nicht. Wir müssen aber einfach die Option abbrechen). Außerdem so klingt für mich auch "the offer associated with the product option", dass der Kunde auch nicht das Produkt (Trikot in diesem Fall) nicht will, was auch nicht unbedingt stimmt (vielleicht hat er sich besser überlegt und will keine zusätzliche Option, oder?)

Vorschlag: Click this button to cancel the product options.

Reply:
Das ist genau der Fall bei CANCEL. Er will das Produkt nicht mehr. Er muss SAVE drücken, damit das Produkt in die Bill list aufgenommen wird